tests/unit/models/test_project.py

```diff
@@ -170,15 +169,15 @@ def test_build_info_from_platforms(incoming, expected):
                 Platform(build_on=[arch], build_for=[arch]),
                 id=arch,
             )
-            for arch in platforms._ARCH_TRANSLATIONS_DEB_TO_PLATFORM
```

Comment:
This contains `powerpc` so I switched to the craft-platforms enum that replaces this.